client/packages/host/src/Admin/DisplaySettings.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
```

Comment:
this is just moved out of `Settings` which was the original file. 
All the new settings were created as their own component, so I'm doing the same with the display settings